spec/latest/json-ld-api/index.html

```diff
@@ -1187,7 +1187,21 @@
         To begin, the <a>active property</a> is set to <code>null</code>,
         and <em>element</em> is set to the <a>JSON-LD input</a>.</p>
 
-      <ol class="algorithm">
+      <p>The algorithm also performs processing steps specific to expanding
+        a <a>JSON-LD Frame</a>. For a <a>frame</a>, the <code>@id</code> and
+        <code>@type</code> properties can accept an array of <a>IRIs</a> or
+        an empty <a>dictionary</a>. The properties of a <a>value object</a> can also
+        accept an <a>array</a> of <a>strings</a>, or an empty <a>dictionary</a>.
```

Comment:
Yes, to be complete, we need to accept an array composed of anything that can be a value of `@value` in the JSON-LD grammar.